Sources/Apodini/Modifier/OperationModifier.swift

```diff
@@ -15,6 +15,19 @@ public enum Operation {
     case update
     /// The associated endpoint is used for a `delete` operation
     case delete
+
+    public var description: String {
+        switch self {
+        case .create:
+            return "create"
+        case .read:
+            return "read"
+        case .update:
+            return "update"
+        case .delete:
+            return "delete"
+        }
+    }
```

Comment:
```suggestion
    public var description: String {
        rawValue
    }
```

How about `String` for the `Operation.RawValue`? E.g. `Operation: String, ...`.

Reply:
Resolved via 6c2b9bf